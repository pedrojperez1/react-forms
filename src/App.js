import './App.css';
import BoxList from "./BoxList";
import boxes from "./boxes"

function App() {
  return (
    <div className="App">
      <BoxList boxes={boxes}/>
    </div>
  );
}

export default App;
