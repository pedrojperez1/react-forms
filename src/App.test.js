import { render, screen } from '@testing-library/react';
import App from './App';

it("renders without going up in smoke", () => {
  render(<App />);
});

it("renders the same each time", () => {
  const {asFragment} = render(<App/>);
  expect(asFragment()).toMatchSnapshot();
})