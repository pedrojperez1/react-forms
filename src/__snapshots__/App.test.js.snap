// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the same each time 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div>
      <div
        class="NewBoxForm"
      >
        <form>
          <input
            class="NewBoxForm-input"
            name="backgroundColor"
            placeholder="Background color"
            type="text"
            value=""
          />
          <input
            class="NewBoxForm-input"
            name="height"
            placeholder="height"
            type="text"
            value=""
          />
          <input
            class="NewBoxForm-input"
            name="width"
            placeholder="width"
            type="text"
            value=""
          />
          <button>
            Add
          </button>
        </form>
      </div>
      <div
        class="Box"
      >
        <div
          class="Box-box"
          style="background-color: pink; height: 100px; width: 100px;"
        />
        <button>
          X
        </button>
      </div>
      <div
        class="Box"
      >
        <div
          class="Box-box"
          style="background-color: salmon; height: 200px; width: 200px;"
        />
        <button>
          X
        </button>
      </div>
      <div
        class="Box"
      >
        <div
          class="Box-box"
          style="background-color: lightblue; height: 50px; width: 80px;"
        />
        <button>
          X
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
